pkg/kv/bulk/kv_buf_test.go

```diff
@@ -51,19 +55,40 @@ func TestKvBuf(t *testing.T) {
 
 	src, totalSize := makeTestData(50000)
 
+	ctx := context.Background()
+	none := mon.NewMonitorWithLimit("none", mon.MemoryResource, 0, nil, nil, 0, 0, nil).MakeBoundAccount()
+	lots := mon.NewUnlimitedMonitor(ctx, "lots", mon.MemoryResource, nil, nil, 0, nil).MakeBoundAccount()
+
 	// Write everything to our buf.
 	b := kvBuf{}
 	for i := range src {
+		size := sz(len(src[i].key) + len(src[i].value))
+		fits := len(b.entries) < cap(b.entries) && len(b.slab)+int(size) < cap(b.slab)
```

Comment:
Guessing we want to pull in https://github.com/cockroachdb/cockroach/pull/78242

Reply:
Indeed, added.